javascript/javascript1/week1/1.js

```diff
@@ -0,0 +1,121 @@
+// Age-IFy excericse
```

Comment:
It is good to have a habit to give a meaningful name to files, I wouldn't say 1.js is the right approach regarding naming